scripts/setup/autoenv.sh

```diff
@@ -27,14 +27,25 @@ fi
 # find HolBA dir for this script
 #####################################################
 
-
-if [ -z "${HOLBA_DIR}" ]; then
+if [[ -z "${HOLBA_DIR}" ]]; then
   HOLBA_DIR=${SETUP_DIR}/../..
   HOLBA_DIR=$(readlink -f "${HOLBA_DIR}")
 fi
 
-if [ ! -d "${HOLBA_DIR}/scripts/setup" ]; then
-  echo "ERROR: HOLBA_DIR not found"
+function is_holba {
+  [[ -d "$1/scripts/setup" ]] \
+      || { echo "bad scripts dir" > /dev/stderr; return 1; }
+  [[ "$(head -n1 "$1/README.md" 2> /dev/null | grep -c '# HolBA')" -eq 1 ]] \
+      || { echo "bad README.md" > /dev/stderr; return 1; }
+  local REMOTE_REGEX='github[.]com[:/]kth-step/HolBA'
+  [[ "$(cd "$1" && git remote -v | grep -c $REMOTE_REGEX)" -gt 0 ]] \
+      || { echo "missing Git remote" > /dev/stderr; return 1; }
+
+  return 0
+}
+
+if ! is_holba "${HOLBA_DIR}"; then
```

Comment:
Just a question. Why not using `[[ ]]` here for consistency?

Reply:
Because 

```bash
[[ ! $(is_holba "${HOLBA_DIR}") ]]  # not sure if we really need $() here

# is equivalent to:

[[ ! -n $(is_holba "${HOLBA_DIR}") ]] # result is always false
# or something like this, I'm not sure:
[[ -n ! $(is_holba "${HOLBA_DIR}") ]]
```
basically, all non-empty strings will be accepted. That's not what we want.

> `man test`:
>
>STRING equivalent to -n STRING

So we would need this: (well, not exactly, this doesn't seem to work)

```bash
[[ ( ! $(is_holba "${HOLBA_DIR}") ) ]]
```

This is quite lengthy and less readable, and I'm not sure about what it does ^^ I guess that I should learn Bash :sweat_smile: 

---

This syntax comes from this SO answer: https://stackoverflow.com/a/5431932

But reading [this one](https://stackoverflow.com/a/5431932) tells that `return 0 # true` and `return 1 # false` should be avoided. Argh Bash...